### ISSUE TITLE
Step assist not working.

### ISSUE THREAD

**Issue by SynexWolf** [rwtema/modjam2018#47]
Forge Version: 14.23.4.2739
Mod version: 1.4
I have advanced to level 8. It says I should have step assist, but it doesn't seem to work when I try to step up a block.

**Comment by Joefish96** [comment]
I'm having the same issue, also the waterwalking just causes me to get stuck on the surface of the water.

**Comment by Sowel** [comment]
Having both step assist and water walking bugs as well as blink not functioning.
I'm using it in the FTB Revelation pack, added it myself.. I'm sure that information is useless but who knows.

**Comment by Drakonas** [comment]
Same issue as @Joefish96 here for both me and my friend in a large modpack.

[3rdForge-1.0.zip](https://github.com/rwtema/Monk-Mod/files/2319912/3rdForge-1.0.zip)
Here's the exported modpack I've made in case the author wants to test.

**Comment by JordanABadger** [comment]
I have the same issue although the water walking issue stopped after a while and works fine now but the step assist issue is still there but another thing I realised is step assist in general wasn't working from other mods aswell, for example Everlasting Abilities' step assist doesn't work nor Botania's Soujourner's sash. So I'm at a loss as to what the issue could be.

**Comment by JordanABadger** [comment]
As a tip to others the water walking issue worked for me after I walked on water and while it was glitching like I was stuck in an invisible block I save and quitted then went back into my world and it was fine and I could walk on water fine.

**Comment by Drakonas** [comment]
Yeah I found it seems to be a client sync issue. It fixes itself when any of us disconnect from the server and reconnect.

**Comment by Chaogomu** [comment]
The water-walking seems to correct itself on disconnect from a server. The step assist does not.

**Comment by Lejoni** [comment]
I had all 3 problems, Stepassist not working, Water walking acting like I was stuck in a block, and blink not working. But save & quit and load again fixed all 3 for me.

**Comment by grydian** [comment]
no matter how many times I login step assist never works even though I have other items that give it to me.  To get it to work I have to put on armor which disables the monk mod :(

**Comment by grydian** [comment]
Removing Thaumic gadgets fixed the problem for me.  Wanted to mention this incase others were still dealing with this issue.

**Comment by darthvader45** [comment]
Yeah, reload fixed it for me too.  Wonder why it doesn't sync between server and client.
